### PR TITLE
Adding a trouble shooting section for setup (SCP-4236)

### PR DESCRIPTION
**Pull request by ehanna4**
When setting up this repo on my local I ran into a couple issues and figured it might be helpful to note them here (like we do for SCP-core set up) to potentially help troubleshoot the same issues in the future for new folks.

### CODE REVIEW

**Review by jlchang — APPROVED** [review]
Love it! I've not run into those particular issues so it's good to know we have docs on how to deal with them if they come up!

**Review by devonbush — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -140,3 +140,14 @@ source env/bin/activate
 ```
 
 See [`ingest_pipeline.py`](https://github.com/broadinstitute/scp-ingest-pipeline/blob/ew-tests-hook/ingest/ingest_pipeline.py) for usage examples.
+
+## Trouble shooting during set up
```
Comment (eweitz) [inline README.md:144]: More in-page-findable:

```suggestion
## Troubleshooting during set up
```

File: README.md [hunk 2]
```diff
@@ -140,3 +140,14 @@ source env/bin/activate
...
+
+If you run into an error like: "... [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed ... " try:
+- Open terminal
+- CD to where python is installed
```
Comment (eweitz) [inline README.md:148]:
```suggestion
- `cd` to where python is installed
```

**Review by eweitz — APPROVED** [review]
Looks good!

For auditability, please add Jira ticket to PR title (even if this PR doesn't suffice for the ticket).

**Pull request merged**